tendrl/commons/flows/service_utils.py

```diff
@@ -0,0 +1,55 @@
+""" This utility can be used to handle(start, stop, etc.) tendrl services.
```

Comment:
@shtripat  @GowthamShanmugam Why we need another service util or did you guys miss this during review?

Could you help @anmolsachan  with re-using and modifying below for this PR

https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/service.py

Reply:
It does disabling of service as well so thought of writing new one. But I feel at least the code can move to https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/service.py

Reply:
Sure. Will make the change